core/web/schema/type/job_run.graphql

```diff
@@ -0,0 +1,12 @@
+type JobRun {
+    outputs: [String]!
+    allErrors: [String]!
+    fatalErrors: [String]!
+    inputs: String!
+    createdAt: Time!
+    finishedAt: Time
+    taskRuns: [TaskRun!]!
+    observationSource: String!
```

Comment:
I don't think we need the observation source.

We could add a separate ticket which allows you to fetch the job associated with the run which will contain the observation source

Reply:
Sure, we could create a ticket to work on that